app/controllers/pages_controller.rb

```diff
@@ -1,4 +1,19 @@
 # Controller to pages entities
 class PagesController < ApplicationController
   def index; end
+
+  def graphics
+    time_now = Time.zone.now
+    @date_from = time_now.to_date.beginning_of_month
+    @date_to = time_now.to_date.next_day
+    transactions_date_range if params[:search].present?
+    @personality = UserPersonality.find(params[:user_personality_id])
```

Comment:
![image](https://user-images.githubusercontent.com/88551298/149034470-e0068805-5f35-484e-8aeb-ae44548c4074.png)
I think it is wrong behaviour

Reply:
In other places all good and i think tomorrow u can pass this

Reply:
Fixed